scripts/dockerfiles/run.sh

```diff
@@ -85,6 +90,58 @@ done
         /var/vcap/jobs/{{ $job.Name }}/bin/run
     {{ end }}
 {{ else }}
+    # Run all the scripts called post-start, if any.
+    psp=$(find /var/vcap/jobs/*/bin -name post-start)
+
+    if [ "X$psp" != X ] ; then
+	# We have post-start scripts to run.
+
+	# We do this by adding a job to the monit configuration
+	# before invoking it. This actually trivial, we simply
+	# put a .monitrc file into the directory /var/vcap/monit/job
+	# and monit will pick it up on start.
+	#
+	# The trick is to make this new job ("PS") dependent on all
+	# the existing jobs (of the role). Because the monit we have
+	# has bugs in its dependency management our job's start
+	# command does the checking manually, parsing the output of
+	# `monit summary`.  We are also using `flock` to guard against
+	# monit starting the job multiple times (due to the checking
+	# itself or the called post-start scripts taking too long).
+	# The start command is a script we arrange to run all the
+	# post-start files. Its last action is setting the monitored
+	# marker file, completing things.
+
+
+	# Create the job script, runs all the post-start scripts
+	# found, then sets the marker.
+	cat > /opt/hcf/post-start.sh <<EOF
+#!/bin/bash
+set -e
+(
+  flock -n 9 || exit 1
+  notyet=\$(monit summary | tail -n+3 |grep -v 'Accessible\|Running'|wc -l)
+  if [ \$notyet -eq 1 ] ; then
+    for fname in ${psp} ; do
```

Comment:
Where would I have to modify the fissile go code to add a fixed post-start.sh ?
Notes
* The `zomega` would also be fixed and can actually go into the `monitrc.erb`.
* This can be seen as a point against, as going to monit with proper dependency testing
   will have to move it out again.
* On the third hand, having it simple now is not really an argument against future complexity.